### PR TITLE
Deploy minio webhook source

### PR DESCRIPTION
**Pull request by pierDipi**
Deploy `minio-webhook-source` as `ksvc` sending events to `predictions-broker` connected using `SinkBinding`

In addition, I'm deploying 2 event displays to display `predictions-broker` events as well as `minio` events (the actual minio events)

```
$ k logs -n ai-demo broker-event-display -f


Context Attributes,
  specversion: 1.0
  type: com.knative.dev.minio.event
  source: pod.ai-demo.minio-webhook-source
  id: 2dcdcd0d-8e13-414a-b540-7243e199c9a9
  time: 2023-09-26T12:56:01.521031Z
Extensions,
  knativearrivaltime: 2023-09-26T12:56:01.529310378Z
Data,
  {"event": "s3:ObjectCreated:Put", "records": [{"eventTime": "2023-09-26T12:56:01.515Z", "bucket": "ai-demo", "object": "minio-webhook-source%2Fksvc.yaml", "etag": "ab66a09dd4aaaad4dd8739c8ae8704e7"}]}
```

### CODE REVIEW

File: infra/openshift-manifests/minio-webhook-source/ksvc.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          ports:
+            - containerPort: 5000
+          env:
+            - name: POD_NAME
+              value: "minio-webhook-source"
+            - name: POD_NAMESPACE
+              value: "ai-demo"
+---
+apiVersion: sources.knative.dev/v1
+kind: SinkBinding
```
Comment (aliok) [inline infra/openshift-manifests/minio-webhook-source/ksvc.yaml:22]: Awesome!

File: infra/openshift-manifests/minio-webhook-source/ksvc.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+metadata:
+  name: minio-webhook-source
+  namespace: ai-demo
+  labels:
+    app: minio-webhook-source
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/pierdipi/minio-webhook-source:v1
```
Comment (aliok) [inline infra/openshift-manifests/minio-webhook-source/ksvc.yaml:12]: Docker doesn't allow us to create an organization and push into the same repository. I wonder if Quay allows it

File: infra/openshift-manifests/minio-webhook-source/ksvc.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+          ports:
+            - containerPort: 5000
+          env:
+            - name: POD_NAME
+              value: "minio-webhook-source"
+            - name: POD_NAMESPACE
+              value: "ai-demo"
```
[the remark this answers is not in the document]
Comment (aliok) [reply infra/openshift-manifests/minio-webhook-source/ksvc.yaml:19]: Not sure what's better... Having podname as the cloud event source, or a hardcoded value

File: infra/openshift-manifests/minio-webhook-source/ksvc.yaml (hunk 2 above)
Comment (pierDipi) [reply infra/openshift-manifests/minio-webhook-source/ksvc.yaml:12]: I use packages and it works well with GH https://github.com/pierDipi?tab=packages&repo_name=sacura

Comment (aliok) [reply infra/openshift-manifests/minio-webhook-source/ksvc.yaml:12]: @pierDipi Nice!

Let me create aGitHub organization and move this repository there. We can push to packages under that org.

File: infra/openshift-manifests/minio-webhook-source/ksvc.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (pierDipi) [reply infra/openshift-manifests/minio-webhook-source/ksvc.yaml]: Done `SOURCE_DECLARATION`

**Pull request closed without merging**